src/renderer/widgets/SendAssetModal/ui/components/TransferForm.tsx

```diff
@@ -345,6 +352,27 @@ export const TransferForm = ({
     }
   };
 
+  const handleMyselfClick = () => {
```

Comment:
As I see in worst case scenario `destinationChainAccounts` is `[]` and cannot be `undefined`.
```typescript
const handleMyselfClick = () => {
  if (destinationChainAccounts.length > 1) {
    setSelectAccountModalOpen(true);
  } else {
    handleAccountSelect(account);
  }
};
```